### ISSUE TITLE
Display metro exit number on metro exits/entries

### ISSUE THREAD

**Issue by rugk** [organicmaps/organicmaps#5110]
**Is your feature request related to a problem? Please describe.**
I'm always frustrated when I am travelling with the metro, have multiple exists and take a wrong (not optimal) one. Instead, I would like to take the correct one.

Signs usually say the street and an exit number, but a non-local and with multiple streets the street name does nor help me much (more below), as such knowing the exit number I need to take would help a lot.

Specifically I was in Paris and this is what is shown in OrganucMaps:
![Screenshot_20230509-115722](https://github.com/organicmaps/organicmaps/assets/11966684/38742b5f-777d-42ae-bf60-335697a86dfc)

**Describe the solution you would like**
Display the number somewhere, either directly on the map near the name or in the panel that expands (though you'd have to check each metro exit then to find out what is the one you want)

**Describe alternatives you have considered**
- How do you solve this issue now with Organic Maps or other apps? → go out "random" or where most people exit (or use another app like Osmand to find the exit) or try the street names, which are also shown on the exits and estimate what is the one they want (that last thing is hard for many streets and crossings like in Paris, so exit numbers are way easier IMHO)
- Attach any examples, screenshots, or screen recordings from other apps that help us to better understand the idea.

Osmand displays it in brackets next to the name:
![Screenshot_20230509-115813](https://github.com/organicmaps/organicmaps/assets/11966684/fa99fdea-69f3-49aa-8fdc-ffe3cfccab48)
![signal-2023-05-09-11-58-35-116](https://github.com/organicmaps/organicmaps/assets/11966684/cf8d76b1-ec4e-4aba-a5eb-1dd54afbc6fd)

**Additional context**
Transportr I used for navigator also cannot really help:
![Screenshot_20230509-120202](https://github.com/organicmaps/organicmaps/assets/11966684/63828851-39ea-4c06-b9e6-599996c49567)
Reported there as https://github.com/grote/Transportr/issues/851

Just saw in drive mode they may already be displayed? https://github.com/organicmaps/organicmaps/issues/5102

This may also help, so is slightly related: https://github.com/organicmaps/organicmaps/issues/5100

**Comment by vbertola** [comment]
Yes please, I noticed this in Japan, where underground metro stations in Tokyo are real mazes and directions are often given in reference to specific exit names. This would also be very useful.

**Comment by vng** [comment]
Concatenate name + ref like we do (or did before) with name + house number?

**Comment by biodranik** [comment]
That can be a quick workaround. Maybe we can do it properly later.
CC @pastk

**Comment by RedAuburn** [comment]
https://github.com/organicmaps/organicmaps/pull/7297 is effectively a template for this
(doesn't display ref on the map though)

**Comment by alnzrv** [comment]
Just checked on version 2026.04.07-8-Google. Is it enough to close the issue?

<img width="640" alt="Image" src="https://github.com/user-attachments/assets/c2050779-68a5-40c2-8ecf-c627e0450cbd" />
